test/app.test.js

```diff
@@ -687,76 +688,27 @@ describe('app', function() {
       });
     });
 
-    it('accepts null dataSource', function() {
-      app.model('MyTestModel', { dataSource: null });
-    });
-
-    it('accepts false dataSource', function() {
-      app.model('MyTestModel', { dataSource: false });
+    it('accepts null dataSource', function(done) {
+      app.model(MyTestModel, { dataSource: null });
+      expect(MyTestModel.dataSource).to.eql(null);
+      done();
     });
 
-    it('should not require dataSource', function() {
-      app.model('MyTestModel', {});
-    });
-  });
-
-  describe('app.model(name, config)', function() {
-    var app;
-
-    beforeEach(function() {
-      app = loopback();
-      app.dataSource('db', {
-        connector: 'memory',
-      });
+    it('accepts false dataSource', function(done) {
+      app.model(MyTestModel, { dataSource: false });
+      expect(MyTestModel.getDataSource()).to.eql(null);
+      done();
     });
 
-    it('Sugar for defining a fully built model', function() {
-      app.model('foo', {
-        dataSource: 'db',
-      });
-
-      var Foo = app.models.foo;
-      var f = new Foo();
-
-      assert(f instanceof app.registry.getModel('Model'));
+    it('does not require dataSource', function(done) {
+      app.model(MyTestModel);
+      done();
     });
 
-    it('interprets extra first-level keys as options', function() {
-      app.model('foo', {
-        dataSource: 'db',
-        base: 'User',
-      });
-
-      expect(app.models.foo.definition.settings.base).to.equal('User');
-    });
-
-    it('prefers config.options.key over config.key', function() {
-      app.model('foo', {
-        dataSource: 'db',
-        base: 'User',
-        options: {
-          base: 'Application',
-        },
-      });
-
-      expect(app.models.foo.definition.settings.base).to.equal('Application');
-    });
-
-    it('honors config.public options', function() {
-      app.model('foo', {
-        dataSource: 'db',
-        public: false,
-      });
-      expect(app.models.foo.app).to.equal(app);
-      expect(app.models.foo.shared).to.equal(false);
-    });
-
-    it('defaults config.public to be true', function() {
-      app.model('foo', {
-        dataSource: 'db',
-      });
-      expect(app.models.foo.app).to.equal(app);
-      expect(app.models.foo.shared).to.equal(true);
+    it('throws error if model typeof string is passed', function(done) {
+      var fn = function() { app.model('MyTestModel'); };
+      expect(fn).to.throw(Error);
+      done();
```

Comment:
@bajtos: Isn't it the `best practice` to have `done()` to indicate end of the test?